src/test/java/uk/gov/pay/directdebit/util/DatabaseTestHelper.java

```diff
@@ -0,0 +1,82 @@
+package uk.gov.pay.directdebit.util;
+
+import org.skife.jdbi.v2.DBI;
+import org.skife.jdbi.v2.util.StringColumnMapper;
+import uk.gov.pay.directdebit.payments.fixtures.PaymentRequestFixture;
+import uk.gov.pay.directdebit.payments.fixtures.TokenFixture;
+
+import java.sql.Timestamp;
+import java.util.Map;
+
+public class DatabaseTestHelper {
```

Comment:
Fixtures looks very nice! but would be even nicer for them to insert themselves so only needed JDBI and not databasehelper (as done in new end to end approach) and therefore keeping everything small and much easily localisable.

Reply:
👍  Will change too as part of the next PR